src/cosmicqc/cli.py

```diff
@@ -0,0 +1,111 @@
+"""
+Setup coSMicQC CLI through python-fire
+"""
+
+import inspect
+import sys
+import types
+from typing import Optional
+
+import fire
+from fire import helptext, inspectutils, value_types
+from fire.core import Display, FireError, _DictAsString, _OneLineResult
+from fire.trace import FireTrace
+
+from . import analyze
+
+
+# referenced from https://github.com/google/python-fire/pull/446
+# to be removed after python-fire merges changes (uncertain of timeline)
+def HasCustomRepr(component: object) -> bool:
+    """Reproduces above HasCustomStr function to determine if component has a
+    custom __repr__ method.
```

Comment:
I might recommend including a reason why this is necessary. What would be the issue if the analyze class didn't have the __repr__ function?

Reply:
Good point - I'll update the docs here. Roughly: without this and the related functions Pandas __repr__ within the CLI fails.